src/System.Runtime.Extensions/tests/Performance/System/Perf.Math.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Xunit.Performance;
+
+namespace System
+{
+    public class Perf_Math
+    {
+        [Benchmark]
+        public void Floor_Decimal()
+        {
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                using (iteration.StartMeasurement())
+                {
+                    for (int i = 0; i < 3000000; i++)
+                    {
+                        Math.Floor(45.2m);
```

Comment:
Note that the Math operations are often treated as intrinsics by the codegenerators. If you call the Math operation with a constant, the codegenerator may just replace it with a constant - without actually calling the method.